core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -268,7 +268,7 @@ oppia.controller('StateResponses', [
               ExplorationHtmlFormatterService.getInteractionHtml(
                 stateInteractionIdService.savedMemento,
                 stateCustomizationArgsService.savedMemento,
-                'testInteractionInput'));
+                'testInteractionInput'), false);
```

Comment:
This seems wrong. Make sure to carefully proofread.

Reply:
Done